getting-started/eclipselink/README.md

```diff
@@ -74,15 +73,15 @@ This example requires `jq` to be installed on your machine.
 5. Then, use the access token in the Authorization header when accessing Polaris:
 
     ```shell
-    curl -v http://127.0.0.1:8181/api/management/v1/principal-roles -H "Authorization: Bearer $POLARIS_TOKEN"
-    curl -v http://127.0.0.1:8181/api/management/v1/catalogs/quickstart_catalog -H "Authorization: Bearer $POLARIS_TOKEN"
+    curl -v http://localhost:8181/api/management/v1/principal-roles -H "Authorization: Bearer $POLARIS_TOKEN"
```

Comment:
For some other scripts we use POLARIS_HOST, like so:
```
env POLARIS_HOST=localhost ./regtests/run.sh
```

Should we consider doing something similar here?